admin/tabs/activity/tracking/ActivityDetail.js

```diff
@@ -54,11 +54,11 @@ const detail = hoistCmp.factory(
                 }),
                 jsonInput({
                     omit: !data.data,
-                    value: data.data,
-                    disabled: true,
-                    height: 100,
+                    value: JSON.stringify(JSON.parse(data.data), null, 2),
```

Comment:
We don't have a mode by which jsonInput will pretty print automatically.

Reply:
Not that I could find, seems like what you would usually want as default behavior - Filed #1923